terraform/main.tf

```diff
@@ -79,16 +79,17 @@ module "postgres_db_development" {
   source = "github.com/LBHackney-IT/aws-hackney-common-terraform.git//modules/database/postgres"
   environment_name = "development"
   vpc_id = data.aws_vpc.development_vpc.id
-  db_name = "mosaic-mirror"
-  db_port  = 5002
-  subnet_ids = data.aws_subnet_ids.development_private_subnets.ids
   db_engine = "postgres"
   db_engine_version = "12.2"
+  db_identifier = "mosaic-mirror"
   db_instance_class = "db.t2.micro"
+  db_name = "mosaic-mirror"
+  db_port  = 5002
+  db_username = "${local.parameter_store}/mosaic-postgres-username-development"
```

Comment:
Is this referencing to an ssm parameter?

Reply:
Can confirm a param exists in SSM with this name, in the Development APIs account.

Looks like line 18 creates the reference to the correct bit of Parameter Store.

Reply:
I've been naming the SSM parameters with the convention `mosaic-api/staging/db_name`, I'm not terribly attached to this so could switch to how this one is named but should pick one or the other. What do you think?

Reply:
I like `mosaic-api/staging/db_name` (or `db-name`). I had to add `/mosaic-api/development/postgres-hostname` for testing credentials with `terraform plan` (well, seeing the error about a missing parameter was a sufficient result there, but wanted to see if `plan` could succeed also) so it looks like that convention is in half in place already.

I'll update `mosaic-postgres-username-development` to match. 👍

Reply:
nice 👍 I can clean up the other parameters which don't match it as well